README.md

```diff
@@ -51,7 +51,7 @@ You can also specify a release (tag), if you do not want the latest code, by usi
 git clone --branch 1.1.0 git@github.com:apache/openwhisk-client-go
 ```
 
-You can also pull the code from a fork of the repository. If you intend to become a Contributor to the project, read the section [Contributing to the project](#contributing-to-the-project) below on how to setup a fork.
+You can also pull the code from a fork of the repository. If you intend to become a Contributor to the project, read the section [Contributing to the project](https://github.com/apache/openwhisk-client-go/blob/master/CONTRIBUTING.md).
```

Comment:
Yes, we should link to the separate CONTRIBUTING.md, but there is some basics here... could you put this back and perhaps link to the separate document at the start of that (i.e., #contributing-to-the-project) section?

Reply:
Updated doc to include contributing to the project section